ruby_on_rails/rspec.md

```diff
@@ -0,0 +1,114 @@
+# Use Rspec
+
+Rspec is a *de-facto* standard and is used in all our projects.
+We love rspec and we strongly suggest you to use it as well.
+Even if is not mandatory, we believe you should discuss with your team the decision of not using it.
+
+* Add the following gems to your Gemfile
+
+```ruby
+group :test do
+  gem 'capybara'
+  gem 'database_cleaner'
+  gem 'selenium-webdriver'
+  gem 'factory_girl_rails'
+  gem 'faker'
```

Comment:
at the moment yes. this is part of further improvements I'd say and we can create an issue